src/test/scala/cognite/spark/v1/TimeSeriesRelationTest.scala

```diff
@@ -936,6 +936,83 @@ class TimeSeriesRelationTest extends FlatSpec with Matchers with SparkTest with
     assert(descriptionsAfterUpdate.length == 5)
   }
 
+  it should "support ignoring unknown ids in deletes" in {
+    val unit = "ignore-unknown-id-test"
+    cleanUpTimeSeriesTestDataByUnit(unit)
+    val dfWithDeletesAsUnit = retryWhile[Array[Row]](
+      spark.sql(s"select * from destinationTimeSeries where unit = '$unit'").collect,
+      df => df.length > 0)
+    assert(dfWithDeletesAsUnit.length == 0)
+
+    // Insert some test data
+    spark
+      .sql(s"""
+              |select "foo" as description,
+              |name,
+              |false as isString,
+              |map("bar", "test") as metadata,
+              |'$unit' as unit,
+              |null as assetId,
+              |false as isStep,
+              |null as securityCategories,
+              |bigint(0) as id,
+              |'delete example' as externalId,
+              |now() as createdTime,
+              |now() lastUpdatedTime,
+              |dataSetId
+              |from sourceTimeSeries
+              |limit 1
+     """.stripMargin)
+      .select(destinationDf.columns.map(col): _*)
+      .write
+      .insertInto("destinationTimeSeries")
+
+    // Check if insert worked
+    val idsAfterInsert =
+      retryWhile[Array[Row]](
+        spark
+          .sql(s"select id from destinationTimeSeries where unit = '$unit'")
+          .collect,
+        df => df.length < 1)
+    assert(idsAfterInsert.length == 1)
```

Comment:
I think we can remove all of this. Just makes the tests slower, and isn't really used nor necessary to test this functionality.